Resources/Prototypes/Roles/Jobs/Wildcards/boxer.yml

```diff
@@ -0,0 +1,32 @@
+- type: job
+  id: Boxer
+  name: "boxer"
+  startingGear: BoxerGear
+  setPreference: false
+  departments:
+  - Civilian
+  icon: "Boxer"
+  supervisors: "the head of personnel"
+  access:
+  - Service
+  special:
+  - !type:AddComponentSpecial
+    components:
+    - type: Boxer
+      modifiers:
+        coefficients: #Remember these only apply to unarmed
+          Blunt: 1.5
+          Slash: 1.5
+          Piercing: 1.5
+
+- type: startingGear
+  id: BoxerGear
+  equipment:
+    jumpsuit: UniformShortsRed
+    back: ClothingBackpackFilled
+    id: BoxerPDA
+    ears: ClothingHeadsetService
+    gloves: ClothingHandsGlovesBoxingRed
+  innerclothingskirt: UniformShortsRedWithTop
+  satchel: ClothingBackpackSatchelFilled
+  duffelbag: ClothingBackpackDuffelFilled
```

Comment:
indentation?

Reply:
Every single job uses that indentation, is it incorrect?

Reply:
Look at StartingGearPrototype.cs
Those are not part of the equipment field.

Reply:
![image](https://user-images.githubusercontent.com/60792108/178987187-36d81b2f-e1e9-4c36-8471-fa7acd52d40d.png)